### ISSUE TITLE
Undefined index: icon

### ISSUE THREAD

**Issue by mateusjunges** [jeroennoten/Laravel-AdminLTE#312]
**AdminLTE version:** 1.23
**Laravel version:** 5.7.28

![image](https://user-images.githubusercontent.com/19756164/61217035-377a6300-a6e5-11e9-84f6-ab9bb975eb64.png)

I solved it publishing the package views and swapping 
```
$item['icon'] or 'circle-o'
```
 with 
```
isset($item['icon'] ? $item['icon'] : 'circle-o'
```

Feels weird since the first one is a blade shorthand to the second.

Related to #47 and #246

**Comment by lacodimizer** [comment]
Hi @mateusjunges ,

it was fixed in version 1.24.1.

**Comment by mateusjunges** [comment]
Ok, thanks. I'll upgrade the package.
